hack/benchmark/time-to-k8s/public-chart/generate-chart.go

```diff
@@ -70,8 +71,8 @@ func readInLatestBenchmark(latestBenchmarkPath string) benchmark {
 		log.Fatal(err)
 	}
 
-	var cmd, api, k8s, dnsSvc, app, dnsAns float64
-	steps := []*float64{&cmd, &api, &k8s, &dnsSvc, &app, &dnsAns}
+	var cmd, api, k8s, dnsSvc, app, dnsAns, cpu float64
+	steps := []*float64{&cmd, &api, &k8s, &dnsSvc, &app, &dnsAns, &cpu}
```

Comment:
The issue with adding `cpu` here is that on line 117 is adds all the steps to the total, so `cpu` is going to be added to the total.

If you remove `cpu` from the steps slice, the only thing you have to do is add an assignment statement after the for loop on line 111 to append the cpu value from `values` to `cpu` instead.

Reply:
Good catch! Thanks! Fixed.